service/controller/v19/templates/cloudformation/guest/nat_gateway.go

```diff
@@ -11,19 +12,20 @@ const NatGateway = `{{define "nat_gateway"}}
         Fn::GetAtt:
         - NATEIP
         - AllocationId
-      SubnetId: !Ref PublicSubnet
+      SubnetId: !Ref {{ .PublicSubnetName }}
       Tags:
         - Key: Name
-          Value: {{ $v.ClusterID }}
-  NATEIP:
```

Comment:
Why did this line get removed? I don't quite get why we get the attribute a few lines above then.

Reply:
`NATEIP` was a hard coded resource name that is now rendered from `{{ .NATEIPName }}`. In practice, resulting CF template will have `NATEIP00`, `NATEIP01` etc.

Reply:
Okay, then line 12 in this file is probably wrong?

Reply:
Yes, good catch! Fixing